packages/format-library/src/text-color/index.js

```diff
@@ -75,7 +75,7 @@ function TextColorEdit( {
 		[ contentRef, value, colors ]
 	);
 
-	const hasColorsToChoose = colors.length || ! allowCustomControl;
+	const hasColorsToChoose = colors.length || allowCustomControl;
```

Comment:
```suggestion
	const hasColorsToChoose = !! colors.length || allowCustomControl;
```

Suggestion (non-blocking): Let's cast the color length value to a Boolean. This ensures that `hasColorsToChoose` is always a boolean.

Reply:
Done, seems it was originally boolean but when lodash was stripped out they left it as the bare `colors.length` without any tests.